### ISSUE TITLE
Test & review the Azure deployment template we got from Microsoft

### ISSUE THREAD

**Issue by ttmc** [bigchaindb/bigchaindb#503]
- [x] Get account set up to use Microsoft Azure
- [x] Initial test the Azure Resource Manager template (JSON) Microsoft sent to us (to install BigchainDB on a VM on Azure)
- [x] Provide feedback to Microsoft

**Comment by ttmc** [comment]
I have a Microsoft account that I could use with Microsoft Azure, but first I asked around to see if we have any (e.g. a corporate account or whatever).

As it happens, Bruce signed up for the [Microsoft BizSpark](https://www.microsoft.com/bizspark#start-one) program to give deals to startup companies some time ago. His bruce@asc... account initiated that and then he (or someone) added four more team members, all of whom could get the deals. (You can only have five team members total.) Two of the team members were since "deleted" but we can't add any more.

Bottom line: currently there are three BigchainDB (the company) -related Microsoft accounts that can get 130 Euros per month of Azure usage for free (until our BizSpark deals expire in about a year): Bruce, Dimi and Trent.
